controller/timeout/timouts.go

```diff
@@ -63,15 +64,15 @@ func (h *timeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	go func() {
 		defer func() {
 			if p := recover(); p != nil {
```

Comment:
@andrewpmartinez I added this because we used to get stackdumps on panics, and don't anymore. Let me know if you have any objections

Reply:
A perfectly logical change. I approve!